src/DBImport.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace TheRealMVP;
+
+class DBImport
+{
+    protected $pdo;
+
+    /**
+     * DBImport constructor.
+     *
+     * Creates new PDO object
+     */
+    public function __construct()
+    {
+        $this->pdo = new \PDO ("mysql:host=db; dbname=TheRealMVP", "root", "password");
```

Comment:
I like how you store the PDO connection as a property, but what I don't like is how you're having to create two PDO connections.

What if the PDO connection was created outside the class, and passed in instead? You could do the same thing for your TeamHydrator allowing you to recycle a single PDO connection.